aion-api/src/main/java/com/aion/dashboard/controllers/Dashboard.java

```diff
@@ -501,16 +503,16 @@ public String viewDashboard() {
 
         }
     }
-
-    // Third Party APIs
+    @Cacheable(CacheConfig.CIRCULATING_SUPPLY)
     @GetMapping(value = "/getCirculatingSupply")
     public String getCirculatingSupply() {
         try {
-            return thirdPartyService.getCirculatingSupply();
+             return circulatingSupplyJPARepository
+                     .findByOrderByTimestampDesc(PageRequest.of(0, 1)).getContent().get(0).
+                             getSupply().setScale(0, BigDecimal.ROUND_HALF_EVEN).toString();
```

Comment:
Recommended to use `RoundingMode.HALF_EVEN` instead

Reply:
FYI, this is on JDK 11, if the server is not using that version of JDK then my bad, go with what is appropriate.

Reply:
yes, we are using JDK11, so I will change